akka-cluster-sharding-typed/src/test/java/jdocs/akka/cluster/sharding/typed/AccountExampleWithEventHandlersInState.java

```diff
@@ -49,7 +48,6 @@ public CreateAccount(ActorRef<OperationResult> replyTo) {
         this.replyTo = replyTo;
       }
 
-      @Override
       public ActorRef<OperationResult> replyTo() {
```

Comment:
we could now reduce some boilerplate by using
```
public final ActorRef<OperationResult> replyTo;
```
instead of getter method

Reply:
Sure, there are two other commands that are using private fields with getters: `PlaceBid` and `StartAuction`. Should I change them to follow the same pattern? Meaning, always using `public final` and no getters?

Reply:
Yes, that’s what we do for message classes